src/com/mambu/apisdk/util/RequestExecutorImpl.java

```diff
@@ -165,6 +141,284 @@ public String executeRequest(String urlString, ParamsMap params, Method method,
 		return response;
 	}
 
+	/*
+	 * Executes a request in order to download content and returns it as a ByteArrayOutputStream
+	 */
+	@Override
+	public ByteArrayOutputStream executeRequest(String urlString, ParamsMap params, ApiDefinition apiDefinition)
+			throws MambuApiException {
```

Comment:
Would this method work with any other API return types?  May be we can add some checks here and throw exceptions to ensure this method is called only with the expected params that support ByteArrayOutputStream output?